### DISCUSSION TITLE
[A-L] (16 Gennaio 2023) Esercizio 7

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#370] [Esami A-L]
Formalizzare le frasi seguenti in un linguaggio proposizionale e determinare se l’ultima proposizione (che inizia con Dunque) è conseguenza logica della precedenti.

Potete usare un metodo a piacere (tavole di verità, risoluzione, ragionamento sulla conseguenza logica ad alto livello):

- Ogni caramella è o morbida o alla menta o rossa.

- Nessuna caramella rossa è alla menta.

- Nessuna caramella morbida è rossa.

- Dunque nessuna caramella alla menta è morbida.

**Comment by cek753** [comment]
R=rossa M=morbida m=menta
P1:  M ∨ m ∨  R
P2:  ¬(R ∧ m)
P3:  ¬(M ∧ R)
P4:  ¬(m ∧ M)

Per verificare se P4 è conseguenza logica basta vedere se ((P1∧P2∧P3)→P4)∈Taut
Con la tabella di verità si capisce che P4 non è conseguenza logica.
